tests/functional/vault/test_config.py

```diff
@@ -80,7 +80,16 @@ def test_vault_reinitialization(guardian, gov, rewards, token, Vault):
     ],
 )
 def test_vault_setParams(
-    chain, gov, guardian, management, vault, rando, getter, setter, val, guard_allowed,
+    chain,
```

Comment:
Seems like some black updated code in this PR

Reply:
This will be rebased on top of https://github.com/yearn/yearn-vaults/pull/310